be/src/runtime/load_channel.cpp

```diff
@@ -275,6 +279,8 @@ std::shared_ptr<TabletsChannel> LoadChannel::get_tablets_channel(int64_t index_i
         auto local_tablets_channel = dynamic_cast<LocalTabletsChannel*>(it->second.get());
         if (local_tablets_channel) {
             local_tablets_channel->incr_num_ref_senders();
+        } else {
```

Comment:
What's the usage of this `else` branch?

Reply:
Tell others that we didn't forget to handle the else branch.